packages/webcert/src/feature/careProvider/ExpandableTableRow.tsx

```diff
@@ -0,0 +1,88 @@
+import { faAngleDown, faAngleUp } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Unit, UnitStatistic } from '@frontend/common'
+import React, { Fragment, useState } from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+import { getUnitStatistics } from '../../store/user/userSelectors'
+
+const StyledArrow = styled(FontAwesomeIcon)`
+  cursor: pointer;
+`
+
+interface ExpandedUnitProps {
+  isExpanded: boolean
+  unit: Unit
+  handleChooseUnit: (event: React.MouseEvent) => void
+  statistics: UnitStatistic
+}
+
+const ExpandedUnit: React.FC<ExpandedUnitProps> = ({ unit, isExpanded, handleChooseUnit, statistics }) => {
```

Comment:
This can be extracted to its own file. Also I think a lot of functionality in this file can be made more generalized. The name of the component is ExpandableTableRow but it has a close connection to the units. That content can be expandable is something that has been used before in the application and will be used in the future. Therefore you could see if you can use an existing component to create the expandable content, or create a new generalized component. I think it would be good to separate displaying the unit information and the exapandable logic so that the expandable component can be used without the unit and the unit information can be shown without being expandable. Maybe you could even separate the unit and the unit statistics into two components.